lib/games.js

```diff
@@ -458,6 +458,14 @@ export const games = {
     },
     release_year: 2007
   },
+  codbo3: {
+    name: 'Call of Duty: Black Ops 3',
+    release_year: 2015,
+    options: {
+      port_query_offset: 2,
```

Comment:
If we have a `port_query_offset`, we'd want to also have `port` or `port_query` (to be explicit about from what we are adding/substracting without searching for the default protocol port).
I've found a guide that lists `27017` as default port and all of the 3 servers I found online were on that port, and the query port seems to be the same, so just `port: 27017` is required here.

Reply:
That's good to know! And thank you for founding the default port.
I've copied this from the `Call of Duty: Modern Warfare 3`, so I'll make sure to take a look at as well:

https://github.com/gamedig/node-gamedig/blob/b481425dbf1d50dcb3e756897d7cc15a46cc04c9/lib/games.js#L477-L484